cadc-vosi/src/main/java/ca/nrc/cadc/vosi/AvailabilityClient.java

```diff
@@ -89,6 +89,11 @@
     private static Logger log = Logger.getLogger(AvailabilityClient.class);
 
     public static final String AVAILABILITY_ENDPOINT = "/availability";
+    public static final String MIN_DETAIL_PARAMETER = "?detail=min";
+    public static final int DEFAULT_CONNECTION_TIMEOUT = 6000;
+    public static final int DEFAULT_READ_TIMEOUT = 12000;
+    public static final int DEFAULT_MAX_RETRIES = 1;
```

Comment:
max retries should really be 0 (as stated in HttpTransfer javadoc); the fact that the numRetries gets incremented before comparing in HttpTransfer is a bug that causes 1 to also be "no retries" but don't rely on that here

Reply:
Okay updated max retries to 0.